superset-frontend/src/katalon/CustomFilter/FilterExtension.tsx

```diff
@@ -146,11 +146,15 @@ function FilterExtension(props: FilterExtensionProps) {
         >
           {visibleFilter.length !== 0 &&
             visibleFilter.map((item, index) => (
-              <div key={index}>{item.element}</div>
+              <div id={item.name} key={index}>
+                {item.element}
+              </div>
             ))}
           {addFilter.length !== 0 &&
             addFilter.map((item, index) => (
-              <div key={index}>{item.element}</div>
+              <div id={item.name} key={index}>
+                {item.element}
+              </div>
             ))}
```

Comment:
check merge main this. I think that we update the logic here
@triquanminh

Reply:
done